src/declaration.d

```diff
@@ -1357,6 +1357,25 @@ extern (C++) class VarDeclaration : Declaration
             storage_class &= ~stc; // strip off
         }
 
+        if (storage_class & STCscope)
+        {
+            StorageClass stc = storage_class & (STCstatic | STCextern | STCmanifest | STCtls | STCgshared);
+            if (stc)
+            {
+                OutBuffer buf;
+                stcToBuffer(&buf, stc);
+                error("cannot be scope and %s", buf.peekString());
```

Comment:
I tend to use the quotes when the sentence makes no grammatical sense otherwise. But when the meaning is clear without the quotes, I prefer not to.